model/src/main/java/com/ge/predix/audit/rest/PredixAuditRequest.java

```diff
@@ -0,0 +1,49 @@
+package com.ge.predix.audit.rest;
+
+public class PredixAuditRequest {
```

Comment:
Please move this, response and AuditRecord classes to integration-test project if they are not used by the service.